src/less/navbar.less

```diff
@@ -17,7 +17,7 @@
   &:extend(.clearfix all);
 
   @media (min-width: @grid-float-breakpoint) {
-    border-radius: @navbar-border-radius;
+    .border-radius(@navbar-border-radius);
```

Comment:
Personally I'm indifferent, the mixins came from bootstrap so at the time it was the "fast" way because there was no work :) 

Given our support matrix I'm not sure that we really need to prefix much, maybe just a couple of the newest properties...